e2e/suite_test.go

```diff
@@ -0,0 +1,181 @@
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	mrand "math/rand"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/Azure/agentbaker/pkg/agent"
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+	"github.com/barkimedes/go-deepcopy"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var cases = map[string]scenarioConfig{
+	"base": {},
+	// "gpu": {
```

Comment:
Do we want to remove this commented code/move it to a separate gpu case (without using it)?

Reply:
uncommented